draft-ietf-quic-recovery.md

```diff
@@ -537,8 +537,13 @@ and otherwise it MUST send an Initial packet in a UDP datagram of at least
 1200 bytes.
 
 The crypto retransmission timer is not set if the time threshold
-{{time-threshold}} loss detection timer is set.  When the crypto
-retransmission timer is active, the probe timer ({{pto}}) is not active.
+{{time-threshold}} loss detection timer is set.  The time threshold loss
+detection timer is both expected to expire earlier than the crypto
+retransmission timeout and be much less likely to spuriously retransmit data.
+The Initial and Handshake packet number spaces are typically going to have a
```

Comment:
I'm not sure what the number of packets in these spaces has to do with the description here about crypto timeout. Why is this sentence here?

Reply:
If the number of packets is small, it's unlikely you'll hit the packet threshold loss detection, making time threshold much more important.

Reply:
I tightened up the text, but feel free to suggest further clarification.